src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/tpt/TptTranslatorTest.java

```diff
@@ -0,0 +1,102 @@
+package fi.otavanopisto.kuntaapi.test.server.unit.tpt;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.ZoneId;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.apache.deltaspike.testcontrol.api.junit.CdiTestRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import fi.metatavu.kuntaapi.server.rest.model.Job;
+import fi.otavanopisto.kuntaapi.server.id.JobId;
+import fi.otavanopisto.kuntaapi.server.id.OrganizationId;
+import fi.otavanopisto.kuntaapi.server.integrations.KuntaApiConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.tpt.TptConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.tpt.TptTranslator;
+import fi.otavanopisto.kuntaapi.server.integrations.tpt.client.model.ApiResponse;
+import fi.otavanopisto.kuntaapi.server.integrations.tpt.client.model.DocsEntry;
+import fi.otavanopisto.kuntaapi.test.AbstractTest;
+
+@RunWith (CdiTestRunner.class)
+public class TptTranslatorTest extends AbstractTest {
+  
+  @Inject
+  private TptTranslator tptTranslator;
+
+  @Test
+  public void testNull() {
+    assertNotNull(tptTranslator);
+    tptTranslator.translateJob(getJobId(), "https://api.example.com", null);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "https://api.example.com" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)